src/libUtils/Logger.cpp

```diff
@@ -27,15 +27,8 @@
 using namespace std;
 using namespace g3;
 
-string MyCustomFormatting(const LogMessage& msg) {
-  string res = string("[") + msg.level();
-
-  for (unsigned int i = msg.level().size(); i < WARNING.text.size(); ++i) {
-    res += " ";
-  }
-
-  res += "]";
-  return res;
+inline string MyCustomFormatting(const LogMessage& msg) {
+  return string("[") + msg.level().substr(0, 4) + "]";
```

Comment:
If msg.level() length less than 4 software will crash

Reply:
Nope, `string::substr` will only return as many characters as available in that case.
Right now INFO is our shortest level string.

Reply:
I see, then no problem.